### PR TITLE
feat(Deezer): Song & Podcast Covers

### PR DESCRIPTION
**Pull request by StrikerFRFX**
**Describe the changes in this pull request:**
Added covers that will display on both Podcasts and Songs 

Discord_Z3Lj8f20wC
Discord_JA6AY8bY5M

### CODE REVIEW

File: websites/D/Deezer/presence.ts [hunk 1]
```diff
@@ -2,7 +2,11 @@ const presence = new Presence({
   clientId: "607651992567021580"
 });
 
-let oldLang: string = null;
+let oldLang: string = null,
+  albumCoverURL: string,
+  albumCoverID: string,
+  podcastCoverURL: string,
+  podcastCoverID: string;
```
Comment (EncryptedDev) [inline websites/D/Deezer/presence.ts:9]: Improper use of camelCase
```suggestion
  albumCoverURL: string,
  albumCoverId: string,
  podcastCoverURL: string,
  podcastCoverId: string;
```

File: websites/D/Deezer/presence.ts [hunk 2]
```diff
@@ -91,6 +97,20 @@ presence.on("UpdateData", async () => {
...
+      ).url;
+
+      if (albumCoverID !== albumID) {
+        albumCoverID = albumID;
+        albumCoverURL = (
+          await fetch(`https://api.deezer.com/album/${albumCoverID}/image`)
+        ).url;
+      }
+
+      presenceData.largeImageKey = cover ? albumCoverURL : "deezer";
```
Comment (EncryptedDev) [inline websites/D/Deezer/presence.ts:113]: `deezer` is already set on L9. Use the nullish operator instead
```suggestion
      presenceData.largeImageKey = cover ?? albumCoverURL;
```

File: websites/D/Deezer/presence.ts [hunk 3]
```diff
@@ -114,12 +134,30 @@ presence.on("UpdateData", async () => {
...
+      if (podcastCoverID !== podcastID) {
+        podcastCoverID = podcastID;
+        podcastCoverURL = (
+          await (
+            await fetch(`https://api.deezer.com/podcast/${podcastCoverID}`)
+          ).json()
+        ).picture;
+      }
+
+      presenceData.largeImageKey = cover ? podcastCoverURL : "deezer";
```
Comment (EncryptedDev) [inline websites/D/Deezer/presence.ts:160]: `deezer` is already set on L9. Use the nullish operator instead
```suggestion
      presenceData.largeImageKey = cover ?? podcastCoverURL;
```

**Review by EncryptedDev — DISMISSED** [review]

File: websites/D/Deezer/presence.ts (hunk 2 above)
Comment (Slowlife01) [reply websites/D/Deezer/presence.ts:113]: This won't work

Comment (StrikerFRFX) [reply websites/D/Deezer/presence.ts:113]: agreed

**Comment by EncryptedDev** [comment]
You changed the variable names, but never changed them where they are assigned, nice.

File: websites/D/Deezer/dist/metadata.json [hunk 4]
```diff
@@ -25,15 +25,12 @@
     "nl": "Luister altijd en overal naar alle nummers waar je van houdt."
   },
   "url": "www.deezer.com",
-  "version": "2.1.2",
+  "version": "2.1.3",
```
Comment (Slowlife01) [inline websites/D/Deezer/dist/metadata.json:28]:
```suggestion
  "version": "2.2.0",
```

**Review by Slowlife01 — APPROVED** [review]
lgtm

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**